README.rst

```diff
@@ -1,6 +1,220 @@
+    synchronisation features are implemented yet.
+
+- The first version of this API doesn't cache any access nor provide any
+  refresh mechanism. If you want to be sure you have the last data available,
+  issue another call.
+
+Here are is an overview of what the API looks like:
+
+.. code-block:: python
+
+    from kintoclient import Bucket
+
+    bucket = Bucket('personal', server_url='http://localhost:8888/v1',
+                    auth=('alexis', 'p4ssw0rd'))
+    todo = bucket.get_collection('todo', create=True)
+
+    records = collection.get_records()
+    for i, record in enumerate(records):
+        record.data.title = 'Todo #%d' %i
+
+    todo.save_records(records)
+
+
+Handling buckets
+================
+
+All operations are rooted in a bucket. It makes little sense for
+one application to handle multiple buckets at once.
+
+The passed `auth` parameter is a `requests <docs.python-requests.org>`_
+authentication policy, allowing you to authenticate with whatever means
+fits you.
+
+By default, Kinto supports Firefox Accounts and Basic authentication policies.
+
+.. code-block:: python
+
+    bucket = Bucket('payments', server_url='http://localhost:8888/v1',
+                    auth=credentials)
+
+    # Passing `create=True` to the bucket will make an HTTP request to
+    # create it.
+    bucket = Bucket('payments', server_url='http://localhost:8888/v1',
+                    auth=credential, create=True)
+
+
+Collections
+===========
+
+A collection is where records are stored.
+
+.. code-block:: python
+
+    # Once the bucket handy, use it to handle collections or groups.
+    collection = bucket.create_collection('receipts')
+
+    # Or get an existing one.
+    collection = bucket.get_collection('receipts')
+
+    # To delete an existing collection.
+    bucket.delete_collection('receipts')
+
+
+Records
+=======
+
+Records can be retrieved from and saved to buckets.
+
+.. code-block:: python
+
+    # Create an empty record
+    record = collection.create_record()
+
+    # You can also pass a python dictionary to represent the record
+    record = collection.create_record(dict(id='XXX', status='done',
+                                           title='Todo #1'))
+
+    # Get all records
+    record = collection.get_all_records()
+    record = collection.get_record(id='89881454-e4e9-4ef0-99a9-404d95900352')
+    collection.save_record(record)
+    collection.save_records([record1, record2])
+    collection.delete_record(id='89881454-e4e9-4ef0-99a9-404d95900352')
+    collection.delete_records([record1, record2])
+
+
+Handling conflicts
+------------------
+
+XXX
+
+Permissions
+===========
+
+ By default, the authenticated user will get read and write access to the
+ manipulated objects. It is possible to change this behavior by passing a dict
+ to the `permissions` parameter.
+
+ .. code-block:: python
+
+    record = collection.create_record(
+        data={},
+        permissions={'read': ['group:groupid']})
```

Comment:
Note that ``group:groupid`` will have to be changed to ``/buckets/{bucket_id}/groups/{groupid}`` is that what you meant?